torchvision/datasets/kitti.py

```diff
@@ -0,0 +1,184 @@
+import os
+from collections import namedtuple
+from typing import Any, Callable, NamedTuple, Optional, Tuple
+from urllib.error import URLError
+
+import pandas as pd
```

Comment:
instead of importing paandas here, let's import it in `__init__` as done for the `Celeba` dataset.